multiversx_sdk_cli/cli_shared.py

```diff
@@ -172,6 +175,28 @@ def prepare_nonce_in_args(args: Any):
         args.nonce = account.nonce
 
 
+def prepare_chain_id_in_args(args: Any):
+    if not args.chain and not args.proxy:
+        raise ArgumentsNotProvidedError("chain or proxy should be provided")
+
+    if args.chain and args.proxy:
+        proxy = ProxyNetworkProvider(args.proxy)
+        chain_id = proxy.get_network_config().chain_id
```

Comment:
Perhaps rename to `fetched_chain_id` (or something similar).

Reply:
renamed to `fetched_chain_id`